同一类下的异常捕获
@ExceptionHandler(value = {ArithmeticException.class})
@ResponseBody
public String handlerException(Exception e) {
        System.out.println(e);
        return "22222";
}
作用域： 同一个类下

不同类的异常捕获
@ControllerAdvice
public class ExceptionHandle  {

    @ExceptionHandler(value = {Exception.class})
    public void test() {
        System.out.println("222222");
    }
}
